### PR TITLE
Update code42-crashplan.rb

### PR DESCRIPTION
**Pull request by kennyb-222**
major version update to 8.x from 7.x

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/code42-crashplan.rb [hunk 1]
```diff
@@ -1,19 +1,20 @@
 cask "code42-crashplan" do
-  version "7.0.3"
-  sha256 "d0d00edc7855a2c1a5282a138c2d398102d8f5f5d59f148787b0fda9d307b9e3"
+  version "8.2.1,1525200006821,77"
```
Comment (vitorgalvao) [inline Casks/code42-crashplan.rb:2]: Are we sure the `1525200006821` and `77` are values that will change?

Comment (kennyb-222) [reply Casks/code42-crashplan.rb:2]: Unknown to me @vitorgalvao, this is the first release using a different version convention. Previous versions did not use this convention.

Comment (vitorgalvao) [reply Casks/code42-crashplan.rb:2]: @miccal We should probably use `colon` here, [until we have a decision](https://github.com/Homebrew/homebrew-cask/issues/95207). If we don’t get rid of `colon`, it’s the correct thing to use. If we do get rid of it, we should have a better alternative than `after_comma.after_comma`.

And we should pay attention on the next update, to make sure all those values indeed change.

Comment (kennyb-222) [reply Casks/code42-crashplan.rb:2]: @vitorgalvao / @miccal, any blockers on getting this resolved?

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**